pkg/controller/installation/installation_controller.go

```diff
@@ -589,6 +591,32 @@ func (r *ReconcileInstallation) preflightChecks(installation *integreatlyv1alpha
 		}
 	}
 
+	if installation.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+		configMap := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      marin3rconfig.RateLimitConfigMapName,
+				Namespace: installation.Namespace,
+			},
+		}
+		if exists, err := resources.Exists(context.TODO(), r.client, configMap); err != nil {
```

Comment:
Could we do something like we do with the clourd-resource-stragies config here: https://github.com/integr8ly/integreatly-operator/blob/master/pkg/controller/installation/bootstrapReconciler.go#L104-L133 where we have a default "development" config unless there is a sku provided rather than the installation returning in cases where the config map is not there?

Reply:
Good idea. I'll add that to the PR